### PR TITLE
Delete old build section

### LINKED ISSUE #623: Clean up old internal build section
The old internal build docs aren't part of the public wiki anymore. We need to move content we still want to use to the introduction-docs

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* deleted networks as it's currently addressed in https://wiki.iota.org/networks/iota-1.5-chrysalis
* deleted tutorials as it's currently addressed in https://wiki.iota.org/tutorials
* deleted exchange-integration as it's currently addressed in https://wiki.iota.org/introduction/guides/exchange#the-iota-wallet-library 
* deleted cryptography.md as it's not very relevant to the current wiki structure

## Links to any relevant issues

resolves #623 

## Type of change

- Documentation Fix

## How the change has been tested
 
wiki was built locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@Dr-Electron , @jlvandenhout , @aleksei-korolev , could you take a look at this whenever you have a moment?

**Comment by aleksei-korolev** [comment]
from the pipeline run:

```
[INFO] Docusaurus found broken links!
Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
Exhaustive list of all broken links found:
- On source page path = /learn/about-iota/data-transfer:
   -> linking to /build/networks/iota-2.0
- On source page path = /learn/contribute-to-wiki/for_tws/structure:
   -> linking to /participate/support-the-network/about-nodes
   -> linking to /build/welcome
- On source page path = /learn/contribute-to-wiki/reference/style/structure:
   -> linking to /participate/support-the-network/about-nodes
   -> linking to /build/welcome
```

**Comment by lucas-tortora** [comment]
> Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Weird, I didn't get those locally. Looking into it.

File: internal/learn/about-iota/data-transfer.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ A definition of which type of payload is part of the message. The node needs to
...
 
-A side note - in the upcoming 2.0 IOTA version, this PoW requirement will become adaptive and is planned to be completely removed for times of normal network loads... Read more about [IOTA 2.0](/build/networks/iota-2.0).
+A side note - in the upcoming 2.0 IOTA version, this PoW requirement will become adaptive and is planned to be completely removed for times of normal network loads... Read more about [IOTA 2.0](https://wiki.iota.org/goshimmer/welcome).
```
Comment (Dr-Electron) [inline internal/learn/about-iota/data-transfer.md:64]:
```suggestion
A side note - in the upcoming 2.0 IOTA version, this PoW requirement will become adaptive and is planned to be completely removed for times of normal network loads... Read more about [IOTA 2.0](/goshimmer/welcome).
```

**Review by Dr-Electron — APPROVED** [review]
LGTM

**Pull request merged**